take-homes/be-take-home.md

```diff
@@ -0,0 +1,48 @@
+# Tea Subscription Service
+
+You will create a Rails API for a Tea Subscription Service.
+
+## Requirements:
+
+At a minimum, you must create:
+
+- An endpoint to subscribe a customer to a tea subscription
+- An endpoint to cancel a customer's tea subscription
+- An endpoint to see all of a customer's subsciptions (active and cancelled)
+
+Beyond the base requirements, include any additional endpoints you'd like, but be mindful of your time. You don't need to go overboard, consider what might be necessary to achieve MVP so that this API is easy to understand and consume by a Frontend Developer.
+
+## Things we'll be looking for:
+
+- A strong understanding of Rails
+- Ability to create restful routes
+- Demonstration of well-organized code, following OOP
+- Test Driven Development
```

Comment:
I appreciate the clear goals here, as well as the hint towards TDD.  Seeing as this is the only time testing is mentioned, I'll be slightly curious to see how many students pick up on it, but am hoping most automatically know to do this.  (at the very least, we could ask them to do it when we give feedback)

Reply:
Yes, I think part of a take-home is being clear on what you are asked to complete. I'm hoping this will really encourage students to read directions carefully. If they do not, then that is definitely an area of feedback.